README.md

```diff
@@ -83,11 +83,45 @@ require"telescope".setup {
       -- Max number of authors to write in the formatted citation
       -- following authors will be replaced by "et al."
       citation_max_auth = 2,
+      -- Use context awareness by default
```

Comment:
Context awareness is disabled by default

Reply:
Fixed in latest commit